### DISCUSSION TITLE
How Do I Get Old Themes Working In The New SD.Next?

### DISCUSSION THREAD

**Discussion by gohan2091** [vladmandic/sdnext#2390] [Q&A] [answered]
I really like themes like huggingface/idspicQstitho/dracula_revamped and huggingface/remilia/Ghostly. These themes have a nice colour scheme, nice readable text and a clean look. How do I get these working in the new SD.Next? 

This is Dracula and it doesn't fill the full width of the page:

![Screenshot 2023-10-23 155049](https://github.com/vladmandic/automatic/assets/127342824/724f13a3-9b61-4bc0-80d7-2e0ad6452731)

and the settings buttons are messed up: ![Screenshot 2023-10-23 155142](https://github.com/vladmandic/automatic/assets/127342824/6a8889db-317e-4337-bb18-ba167835a62d)

The few themes that work I don't like. Invoked theme is tolerable but would love to get some of the older ones working.

**Comment by vladmandic** [comment]
they are working, that is what author of theme made it look.
previously 3rd party themes were applied on top of sdnext theme. now they are applied on base, so 3rd party theme can define anything.
if you don't like how 3rd party themes look, you can override any behavior by creating your `user.css` and adding css code.
(increasing width would be a single line)

do i like how most 3rd party themes look? no.
but i cannot spend time "optimizing" 3rd party themes.

**Reply by gohan2091** [reply]
I get a message "Using 3rd party theme which is not optimized for SD.Next". Unfortunately my knowledge in coding and .css is virtually nil. Would you be willing to help me fix the width so I can use some of these themes in a more pleasing way? What would this single line say in the user.css (which I can't seem to find at the moment) to get the theme to use the full width of the page?

**Comment by gohan2091** [comment]
If anyone can share with me what single line I need to add to my user.css to get the 3rd party themes using 100% width of the page please let me know.

**Reply by vladmandic** [reply, marked as answer]
create `user.css` with content:
```css
.gradio-container { max-width: unset !important; }
```

**Reply by gohan2091** [reply]
Amazing! Thanks so much. That worked perfectly. Many thanks. You have really helped me a ton!
